tech-specs/src/explore/design/dkg.md

```diff
@@ -1 +1,154 @@
-# Distributed key generation gossip
+# Distributed key generation
+
+Here we give an overview of how the DKG protocol works with the ledger and 
+the roles of each node in the network per this protocol. This portion of the 
+book focuses solely on the production of a new public encryption key via the
+DKG scheme. It does not cover encryption, decryption, or validation of transactions.
+
+Some parts of the DKG protocol may be done through the gossip layer or through
+off-chain communication. However, the entire DKG cannot exist solely in the 
+gossip layer as consensus on the final shared secrets must be achieved. This 
+will require posting information on chain.
+
+## Starting the Protocol
+
+To initializing the protocol to generate a new key, the  current block 
+proposer proposes a new instance. This would be posted on the blockchain and
```

Comment:
clarify what you mean by "new instance" (new instance of the DKG) h/t @Acentelles

Reply:
I think it's maybe better to have the DKG process start at a specific block height; for example, once block 400000*i is reached, then everyone knows that a new DKG phase begins synchronously (without any message or proposal)

Reply:
That was a

> I think it's maybe better to have the DKG process start at a specific block height; for example, once block 400000*i is reached, then everyone knows that a new DKG phase begins synchronously (without any message or proposal)

That actually was a question that I had, namely if it had to be initiated by a validator or it would be scheduled at regular intervals.

Reply:
But I will take the suggestion from @joebebel for now

Reply:
As a practical matter, I think the most important thing is to align the DKG's generated key lifetime with the staking epoch, so the protocol should enforce that the DKG is on the same schedule as staking changes